src/storage/hummock_trace/src/collector.rs

```diff
@@ -70,7 +71,8 @@ fn set_should_use_trace() -> bool {
 
 /// Initialize the `GLOBAL_COLLECTOR` with configured log file
 pub fn init_collector() {
-    tokio::spawn(async move {
+    let rt = Runtime::new().unwrap();
```

Comment:
I am not sure whether the runtime will be dropped after returning from the method. Better to make it a static or global variable.